src/systems/diff_drive/DiffDrive.cc

```diff
@@ -139,6 +143,9 @@ class ignition::gazebo::systems::DiffDrivePrivate
   /// \brief Last target velocity requested.
   public: msgs::Twist targetVel;
 
+  /// \brief Last enable/disable requested.
+  public: msgs::Boolean enable;
```

Comment:
why are you using here `msgs::Boolean` instead of a boolean ?

Reply:
True, it could be a bool, I guess it is because I started with a direct copy from the subscription callback message as for DiffDrivePrivate::OnCmdVel. Do you want me to change it ?

Reply:
yes please